deployment/components/topology-updater/topologyupdater-mounts.yaml

```diff
@@ -26,7 +29,13 @@
   - name: nfd-topology-updater-conf
     mountPath: "/etc/kubernetes/node-feature-discovery"
     readOnly: true
+  - name: kubelet-state-files
+    mountPath: /host-var/lib/kubelet
+    readOnly: true
 
 - op: add
   path: /spec/template/spec/containers/0/args/-
   value: "-podresources-socket=/host-var/lib/kubelet/pod-resources/kubelet.sock"
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: "-kubelet-state-dir=/host-var/lib/kubelet"
```

Comment:
This is the default so we wouldn't actually need this. [ I think same applies to `-podresources-socket` above but we don't need to touch that here in this PR ]

Reply:
Right! Thanks!